src/epub/text/act-2.xhtml

```diff
@@ -895,7 +895,18 @@
 						</tr>
 						<tr>
 							<td epub:type="z3998:persona">Launcelot</td>
-							<td>I will go before, sir. Mistress, look out at window, for all this, There will come a Christian boy, will be worth a Jewess’ eye. <i epub:type="z3998:stage-direction">Exit.</i></td>
+							<td>
```

Comment:
This formatting is incorrect. It should be formatted in a `<td epub:type="z3998:verse">` block. See any other such blocks in this ebook, for example act-2.xhtml line 30.